app/Request.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App;
+
+class Request
+{
+
+	protected $query;
+	protected $input;
+	protected $server;
+
+	public function __construct()
+	{
+		$this->query = $_GET;
+		$this->input = $_POST;
+		$this->server = $_SERVER;
+	}
+
+	public function get($key = null, $default = null)
```

Comment:
Why not to replace ``$_GET``, ``$_POST``with ``$_REQUEST``?

Reply:
personally, to avoid confusing, sometimes you might have the same key in both `$_GET` and `$_POST`, if you use `$_REQUEST` i imagine one of them will be overwritten.

Reply:
Yep, ``$_POST`` will override ``$_GET`` but yeah both are legit anyways